### PR TITLE
Remove . from "Test.Adapter"

### PR DESCRIPTION
**Pull request by vlascik**
I believe the dot here was an error?

### CODE REVIEW

**Comment by rwjblue** [comment]
No, it seems correct to me as it is referencing `Ember.Test.Adapter`.

**Comment by vlascik** [comment]
This would then lead to `import Test.Adapter from "@ember/test/adapter"`, which won't work.

**Comment by rwjblue** [comment]
Then the codemod and poly fill plugin would have to be updated, there are a number nested globals like this...

**Comment by vlascik** [comment]
Hm the only similar one I can see in this RFC is `Ember.RSVP.Promise` which is then generated as `import { Promise } from "rsvp"`, as it is defined in `globals.json` as  `"RSVP.Promise": ["rsvp", "Promise"]`.

Maybe the data for this should be changed from `"Test.Adapter": ["@ember/test/adapter"],` to 
`"Test.Adapter": ["@ember/test/adapter", "Adapter"],` then?

I noticed this while doing https://github.com/ember-cli/ember-rfc176-data/pull/25, I mean, I can work around it, but the import example in README is broken as well.

File: globals.json [hunk 1]
```diff
@@ -143,7 +143,7 @@
   "String.loc": ["@ember/string", "loc"],
   "String.underscore": ["@ember/string", "underscore"],
   "String.w": ["@ember/string", "w"],
-  "Test.Adapter": ["@ember/test/adapter"],
+  "TestAdapter": ["@ember/test/adapter"],
```
Comment (Turbo87) [inline globals.json]: I think we have `["@ember/test/adapter", null, "TestAdapter"]` for these cases 🤔

**Comment by vlascik** [comment]
I'm going to close this, should I create a new PR to change globals.json entry for this to `"Test.Adapter": ["@ember/test/adapter", "Adapter"]`?

**Comment by Turbo87** [comment]
no, because `["@ember/test/adapter", "Adapter"]` is wrong

**Pull request closed without merging**